rel-eng/tito.props

```diff
@@ -530,3 +530,8 @@ whitelist = katello
   rubygem-runcible
   rubygem-typhoeus
   rubygem-zest
+
+
+[foreman-plugins-nightly-rhel7]
```

Comment:
please add it to the `foreman-plugins-1.24-rhel7` section instead, roughly line 333